integration/localnet/bootstrap.go

```diff
@@ -431,6 +463,7 @@ func prepareAccessService(container testnet.ContainerConfig, i int) Service {
 
 	service.Ports = []string{
 		fmt.Sprintf("%d:%d", AccessAPIPort+i, RPCPort),
+		fmt.Sprintf("%d:%d", AccessAPIPort+(i+1), SecuredRPCPort),
```

Comment:
I think this will overlap ports with >1 AN
```suggestion
		fmt.Sprintf("%d:%d", AccessAPIPort+2*i, RPCPort),
		fmt.Sprintf("%d:%d", AccessAPIPort+(2*i+1), SecuredRPCPort),
```